models/account.model.ts

```diff
@@ -31,6 +31,11 @@ const AccountSchema = new Schema<IAccount>({
         type: String,
         required: true
     },
+    status:{
+        type: String,
+        required: true,
+        enum: ['Active', 'Pending Activation', 'Suspended', 'Deactivated', 'Deletedw'],

```

Comment:
it would be best if you created this enum separately and then mention it here because you will need this at other places as well.